v3/util/etcdctl-auth-root.sh

```diff
@@ -0,0 +1,71 @@
+
+#!/bin/bash
+
+source /etc/environment
+
+#Disable authentication
+
+#curl  -L http://127.0.0.1:2379/v2/auth/enable -XDELETE
+
+etcdctlrootusername=$(etcdctl get /etcdctl/config/root-user)
+etcdctlrootpassword=$(etcdctl get /etcdctl/config/root-password)
+etcdctletcdreadusername=$(etcdctl get /etcdctl/config/read-user)
+etcdctletcdreadpassword=$(etcdctl get /etcdctl/config/read-password)
+etcdctletcdreadwriteusername=$(etcdctl get /etcdctl/config/write-user)
+etcdctletcdreadwritepassword=$(etcdctl get /etcdctl/config/write-password)
```

Comment:
Nitpick, but can we replace these with uppercase equivalents and drop the etcdctl?

```
ROOT_USERNAME
ROOT_PASSWORD
READ_USERNAME
READ_PASSWORD
WRITE_USERNAME
WRITE_PASSWORD
```

Then update occurrences below.